docs/config.rst

```diff
@@ -1,11 +1,606 @@
 Configuring CumulusCI
 =====================
 
-cumulusci.yml
--------------
+CumulusCI merges multiple `YAML <https://yaml.org/>`_ files that allow for configuration to occur at several distinct scopes. All of these files have the same name, ``cumulusci.yml``, but live in different locations on the file system. (See `configuration scopes`_ for further detail.)
 
-Global overrides
-----------------
+The `universal cumulusci.yml <https://github.com/SFDO-Tooling/CumulusCI/blob/master/cumulusci/cumulusci.yml>`_ file contains the default configuration stored within CumulusCI itself. This is the location of the standard library of tasks and flows that come with CumulusCI.
 
-Local overrides
----------------
+The project ``cumulusci.yml`` file is located in the ``project`` diretory. You can override aspects of the universal ``cumulusci.yml`` file here by defining project dependencies, configuring new tasks and flows, customizing standard tasks and flows to meet your project's needs, and so much more! 
+
+    .. Important:: When ``cumulusci.yml`` is referenced in this documentation, it refers to the project ``cumulusci.yml`` file unless otherwise specified.
+
+
+
+``cumulusci.yml`` Structure
+---------------------------
+
+There are six top-level sections of a ``cumulusci.yml`` file.
+
+* ``project``: This section is largely generated by running ``cci project init``. It contains information pertaining to the project's package information (if any) and GitHub information.
+
+* ``tasks``: This section defines the tasks that are available to run in your project. See `task configurations`_ for more info on the types of configurations that can be made in this section.
```

Comment:
```suggestion
* ``tasks``: Defines the tasks that are available to run in your project. For configuration options in this section, see `task configurations`_.
```
I suggest 1) simpler wording, and 2) swapping the order of the cross-reference to another part of the doc and the reason the reader would click through to that other part. If you frontload the reason, the reader can immediately assess whether the cross-reference is relevant to them. If it isn't relevant, they can skip over reading those words.